3_visualize/src/merge_plot_legend.R

```diff
@@ -16,14 +16,21 @@ merge_plot_legend <- function(main_plot, legend){
     draw_plot(legend_updated, x = 0.83, y = 0.1, width = 0.08, height = 0.8)+
     # and annotate temperature threshold
     draw_label(expression('75'~degree*'F'), colour = "orangered", x = .95, y = .75, size = 8)+
-    annotate("segment", xend = 0.903, x = 0.92, yend = 0.62, y = 0.738, 
+    annotate("segment", xend = 0.903, x = 0.92, yend = 0.62, y = 0.69, 
              arrow = arrow(length = unit(0.1, "cm")), colour = "orangered")+
     draw_label("threshold", x = .96, y = .705, size = 6, colour = "orangered")+
     # and annotate mean
-    draw_label("Mean", colour = "darkgreen", x = 0.95, y = 0.6, size = 8)+
-    annotate("segment", xend = 0.903, x = 0.92, yend = 0.56, y = 0.56, 
-             arrow = arrow(length = unit(0.1, "cm")), colour = "darkgreen")
+    draw_label("Mean", colour = "darkgreen", x = 0.95, y = 0.545, size = 7)+
+    annotate("segment", xend = 0.903, x = 0.92, yend = 0.545, y = 0.545, 
+             arrow = arrow(length = unit(0.1, "cm")), colour = "darkgreen")+
+    # and 90% CI
+    draw_label("{", x = 0.9, y = 0.47, size = 20, angle = 180, color = "cadetblue3")+
+    draw_label("+/- 90% CI", colour = "cadetblue3", x = 0.95, y = 0.47, size = 7)+
+    # and complete values
+    draw_label("     Predicted \nvalues", x = 0.93, y = 0.3, 
+               size = 6, color = "cornsilk3")
```

Comment:
I like this! It's adding to the legend for both plots though, so we may want a way to turn it on or off depending on the chart

Reply:
I added toggle to turn on/off the brown background. `show_all_predicted` (logical) does the trick. Here's with it on:
![daily_interval](https://user-images.githubusercontent.com/19958841/174322503-2f6d62b3-5322-450f-9ed6-6075064d21a3.png)

And with it off:
![daily_interval](https://user-images.githubusercontent.com/19958841/174322394-84064f57-022f-4f20-8487-2611e0f34e5d.png)